packages/wdio-browserstack-service/src/service.ts

```diff
@@ -110,6 +111,17 @@ export default class BrowserstackService implements Services.ServiceInstance {
         // added to maintain backward compatibility with webdriverIO v5
         this._browser = browser ? browser : globalThis.browser
 
+        // Healing Support:
+        if (!isBrowserstackSession(this._browser)) {
```

Comment:
This function is not currently multi remote compatible

Reply:
Updated this to use `shouldAddServiceVersion()` function itself